packages/component-library/stories/AccessibilityGuidelinesStyle.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';  
+import PropTypes from 'prop-types';
+// import { storiesOf } from '@storybook/react';
+// import { checkA11y } from '@storybook/addon-a11y';

```

Comment:
The commented out imports should be removed across all of the new files.

Reply:
Done.  I also removed them from Welcome.js, which is the file I copied code from to create the new files.